trulens/visualizations.py

```diff
@@ -454,18 +455,269 @@ def _get_hotcold(self):
         return ListedColormap(hotcold, name='hotcold')
 
 
+class Visualizer3D(object):
```

Comment:
instead of copy pasting, can we do inheritance and modular methods?

Reply:
Yes this is the plan - sorry for the late reply I started this just for experimentation so earlier I made this change without. 

I'll have a starter notebook  published and reviewed, and for sure refactor this part.